pkg/plugin/subcommand.go

```diff
@@ -92,3 +92,7 @@ type CreateWebhookSubcommand interface {
 type EditSubcommand interface {
 	Subcommand
 }
+
+type CreateSubcommand interface {
+	Subcommand
+}
```

Comment:
Please, revert this change. We have the interface: 

```go
// CreateAPISubcommand is an interface that represents a `create api` subcommand
type CreateAPISubcommand interface {
	Subcommand
	RequiresResource
}
```

So that we do not want another interface with another name to achieve the same result.
Your plugin must implement CreateAPISubcommand and not  CreateSubcommand 
 
It is a blocker for us to merge this one.

/hold